ktor-network/common/src/io/ktor/network/selector/SocketBase.kt

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2014-2025 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.network.selector
```

Comment:
JIC: probably the package should be `io.ktor.network.sockets` :)

Reply:
Yes, thanks!